oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
At the very least is there a way I can put a test in tests/contrib and then tell travis to ignore it?